Doc/whatsnew/3.11.rst

```diff
@@ -523,6 +523,9 @@ CPython bytecode changes
 * Replaced :opcode:`JUMP_IF_NOT_EXC_MATCH` by :opcode:`CHECK_EXC_MATCH` which
   performs the check but does not jump.
 
+* Replaced :opcode:`JUMP_IF_NOT_EG_MATCH` by :opcode:`CHECK_EG_MATCH` which
```

Comment:
A small thought. Since EGs and all the accompanying opcodes are new for 3.11, these lines document changes that took place between 3.10 and 3.11. For those who never tried the alphas this references the removal of an opcode that never existed (outside of alphas).
Would it make more sense to just remove the previous references to `JUMP_IF_NOT_EG_MATCH` and just mention that this opcode was added?

Reply:
JUMP_IF_NOT_EG_MATCH was documented in the release notes for the 3.11 alpha versions.

Reply:
Aha, they are in the changelog, thanks.